include/common/queue_modes.h

```diff
@@ -0,0 +1,6 @@
+namespace grppi{
```

Comment:
The definition of the enum class has been moved to mpmc_queue.h. Aditionally, the include of this header file has been moved form the common header file to the execution model headers files.